nodejs/deploy

```diff
@@ -37,7 +37,36 @@ nvm install ${NODE_VERSION}
 rm -f ~/.nvm_bin
 ln -s $NVM_BIN ~/.nvm_bin
 
-if [ -f ${CURRENT_DIR}/package.json ]; then
-	pushd $CURRENT_DIR && npm install --production
-	popd
+
+# yarn install
+download_url="https://yarnpkg.com/latest.tar.gz"
```

Comment:
I believe that install yarn using npm should be a better way. It's a simple way and should works with private npm repositories.